back/spec/services/invites/service_spec.rb

```diff
@@ -8,7 +8,7 @@
 
   before do
     settings = AppConfiguration.instance.settings
-    settings['core']['locales'] = %w[fr en nl]
```

Comment:
For some reason, it didn't cause any validation issues even though `fr` and `nl` are not valid locales. The new version started to raise the exceptions as expected, so I fixed but locales, but I didn't delve deeper into the matter.